js/main.js

```diff
@@ -1,6 +1,100 @@
 /* global data */
 /* exported data */
 
+var entriesListEl = document.querySelector('#entries-list');
+
+function createEntry(entry) {
+  var liEl = document.createElement('li');
+  var divImgEl = document.createElement('div');
+  var imgEl = document.createElement('img');
+  var divEl = document.createElement('div');
+  var ptitleEl = document.createElement('p');
+  var ptextEl = document.createElement('p');
+  var editIconEl = document.createElement('i');
+
+  liEl.setAttribute('class', 'row');
+  liEl.setAttribute('data-entry-id', entry.ID);
+  divImgEl.setAttribute('class', 'column-half');
+  imgEl.setAttribute('src', entry.url);
+  divEl.setAttribute('class', 'column-half position-relative');
+  ptitleEl.setAttribute('class', 'title');
+  ptitleEl.textContent = entry.title;
+  ptextEl.setAttribute('class', 'text');
+  ptextEl.textContent = entry.notes;
+  editIconEl.setAttribute('class', 'fa-solid fa-pencil');
+
+  liEl.appendChild(divImgEl);
+  divImgEl.appendChild(imgEl);
+  liEl.appendChild(divEl);
+  divEl.appendChild(ptitleEl);
+  divEl.appendChild(ptextEl);
+  divEl.appendChild(editIconEl);
+  return liEl;
+}
+
+function homeView() {
+  data.view = 'entries';
+  var entriesDivEl = document.querySelector('#entries-container');
+  entriesDivEl.setAttribute('class', 'container');
+
+  var entriesFormEl = document.querySelector('#entries-form-container');
+  entriesFormEl.setAttribute('class', 'hidden');
+}
+
+function formView() {
+  data.view = 'entry-form';
+  var entriesDivEl = document.querySelector('#entries-container');
+  entriesDivEl.setAttribute('class', 'hidden');
+
+  var entriesFormEl = document.querySelector('#entries-form-container');
+  entriesFormEl.setAttribute('class', 'container');
+}
+
+function pageLoad(dataview) {
+  if (data.view === 'entries') {
+    homeView();
+  } else if (data.view === 'entry-form') {
+    formView();
+  }
+}
+
+function contentLoaded(event) {
+  for (var i = 0; i < data.entries.length; i++) {
+    var addingToList = createEntry(data.entries[i]);
+
+    entriesListEl.appendChild(addingToList);
+  }
+
+  pageLoad(data.view);
+
+}
+
+document.addEventListener('DOMContentLoaded', contentLoaded);
+
+var newEntryButton = document.querySelector('.new-button');
+
+function viewSwap(event) {
+  data.view = 'entry-form';
+  event.preventDefault();
+
+  formView();
+  formElement.children[1].children[0].childNodes[1].setAttribute('src', 'images/placeholder-image-square.jpg');
+  formElement.childNodes[1].childNodes[1].childNodes[1].textContent = 'New Entry';
+  titleEl.value = '';
+  urlEl.value = '';
+  notesEl.value = '';
```

Comment:
You can use the `reset` method on the form rather than resetting each of these manually.